examples/coappearances.flatdata

```diff
@@ -0,0 +1,92 @@
+/**
+ * JSON Schema:
+ * {
+ *     "characters": {
+ *         "character_id": {             // two capital letters
+ *             "name": "String",
+ *             "nickname": "String",     // optional
+ *             "description": "String",  // optional
+ *             "relation": {             // optional
+ *                 "kind": "String",
+ *                 "to": "character_id or array of two character_ids"
+ *         }
+ *     },
+ *     "coappearances": [
+ *         {
+ *            "a": "character_id",
+ *            "b": "character_id",
+ *            "chapters": [
+ *                "#.##", ...             // # is a digit
+ *            ]
+ *         }
+ *     ]
+ * }
+ */
+namespace coappearances {
+
+struct Meta {
+    title_ref : u32 : 32;
+    author_ref : u32 : 32;
+}
+
+struct Character {
+    name_ref : u32 : 32;
+}
+
+struct Coappearance {
+    a_ref : u32 : 16;
+    b_ref : u32 : 16;
+    count : u32 : 16;
+    first_chapter_ref: u32 : 16;
+}
+
+struct Nickname {
+    ref: u32 : 32;
+}
+
+struct Description {
+    ref: u32 : 32;
+}
+
+struct UnaryRelation {
+    kind_ref: u32 : 32;
+    to_ref: u32 : 16;
+}
+
+struct BinaryRelation {
+    kind_ref: u32 : 32;
+    to_a_ref: u32 : 16;
+    to_b_ref: u32 : 16;
+}
+
+struct Chapter {
+    major: u8 : 4;
+    minor: u8 : 7;
+}
+
+// @bound_implicitly( characters: vertices, vertices_data )
```

Comment:
We never thought of that... "Bound Implicitly"  meant binding two vectors together, as a columnar way to store a single structure.

Reply:
Was done in #18.